velodrome/transform/influx.go

```diff
@@ -56,6 +65,11 @@ func (config *InfluxConfig) CreateDatabase(tags map[string]string) (*InfluxDB, e
 		return nil, err
 	}
 
+	err = dropSeries(client, measurement, config.DB, tags)
```

Comment:
That's not ideal. How often do you restart a transform, and how long does it take to regenerate the series?